mesop/runtime/runtime.py

```diff
@@ -33,6 +34,7 @@ class PageConfig:
   title: str
   stylesheets: list[str]
   security_policy: SecurityPolicy
+  cors: CORS
```

Comment:
Instead of a separate parameter in PageConfig, I would put CORS into SecurityPolicy (i.e. `allowed_cors_origins`) since CORS is security-related